project/Dependencies.scala

```diff
@@ -20,6 +21,7 @@ object Dependencies {
     scalaVersion := crossScalaVersions.value.head,
     scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.13.4"),
     scalaTestVersion := "3.0.0",
+    specs2Version := "3.8.5",
```

Comment:
It [looks like](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.specs2%22%20AND%20a%3A%22specs2-core_2.12%22) we have to use `3.8.6` to have Scala 2.12 supported.